### DISCUSSION TITLE
Queue Menu Disappeared!

### DISCUSSION THREAD

**Discussion by jafarscharm** [comfyanonymous/ComfyUI#3174] [Q&A]
ComfyUI Queue Menu disappeared, tried clearing browser cache, tried different browsers like chrome, firefox, edge. Tried zoom in/zoom comfyui and browser's, uninstalled recent nodes. when comfyui starting in browser i can see the menu for fraction of seconds then it disappearing every time.

https://github.com/comfyanonymous/ComfyUI/assets/54742441/1d2998cd-7cbc-4838-8798-df47c119ad4d

**Comment by joshgura** [comment]
I still have a queue button, but I noticed there's no longer a button to view the queue after updating to the most recent comfy. the settings menu search function has no reference to  'queue' or 'batch' either so i can't search for the parameter.

edit: sorry about the irrelevant post, in my case the letter Q showed me where the queue moved after the most recent update. the way i'm updating comfy is to clone the github, and in my old installed ComfyUI folder, rename each folder one to foldername_old by looking on the githup page for the release date. (also don't forget to check for new files and folders on the github clone) if it's new, i just replace the folders, and updated without going through the whole install process. of course folders with custom things in them, i just grabbed the default files which were only a few, and replaced my old ones. i still have the _old renamed files and folders but i think i can just delete them. i did this stuff because comfy refused to update by itself and had to try manually. so maybe there's an old file or two in your install or a broken configuration.

**Comment by ltdrdata** [comment]
This issue is no longer valid, so I will close it.
